bundles/framework/admin-announcements/instance.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @class Oskari.framework.bundle.admin-announcements.AdminAnnouncementsBundleInstance
+ *
+ * Main component and starting point for the admin-announcements functionality.
+ *
+ * See Oskari.framework.bundle.admin-announcements.AdminAnnouncementsBundleInstance for bundle definition.
+ */
+Oskari.clazz.define('Oskari.framework.bundle.admin-announcements.AdminAnnouncementsBundleInstance',
+
+    /**
+     * @method create called automatically on construction
+     * @static
+     */
+    function () {
+        var conf = this.getConfiguration();
+        conf.name = 'admin-announcements';
+        conf.flyoutClazz = 'Oskari.framework.bundle.admin-announcements.Flyout';
+    }, {
+
+        afterStart: function () {
+        }
```

Comment:
Do you need to define afterStart() for DefaultExtension to work? I'm not sure so if you do it's ok. If it's not required for overriding something in the DefaultExtension we can remove the empty impl.

Reply:
Yes, it appears that you need to have afterStart() for it to work. At least my bundle did not f.ex. create the tile at all of this was not present.